src/sidebar/components/Thread.js

```diff
@@ -12,6 +12,91 @@ import ModerationBanner from './ModerationBanner';
 
 /** @typedef {import('../helpers/build-thread').Thread} Thread */
 
+/**
+ * Render a header for a hidden top-level thread.
+ *
+ * Every ThreadCard should have a header of some sort representing its top-level
+ * thread. When a thread is visible, an Annotation is rendered which in turn
+ * renders an AnnotationHeader. But when a thread is hidden, Annotation is not
+ * rendered: use this header instead for hidden top-level threads.
+ *
+ * @param {object} props
+ *   @param {Thread['annotation']} props.annotation
+ *   @param {number} props.replyCount
+ *   @param {boolean} props.threadIsCollapsed
+ */
+function HiddenThreadCardHeader({ annotation, ...restProps }) {
```

Comment:
This sub-component existed, but I've renamed it and moved it top top of module as our conventions to this point have tended towards sub-components above default-exported component.